apps/datahub-e2e/src/support/commands.ts

```diff
@@ -128,3 +130,30 @@ Cypress.Commands.add('clearFavorites', () => {
       })
     })
 })
+
```

Comment:
Good idea to add the fixtures! Seems like a good alternative as we use the gn-ui dump for testing and don't need changes upstream in this case.

Reply:
I did not know about the dump, is there any doc about this? Maybe I could have used this dump to have data in filters on local dev instead of configuring the proxy to call the migration api.

Reply:
Currenlty we use the same backend for testing as gn-ui: https://github.com/camptocamp/mel-dataplatform/blob/main/README.md#tests-densemble-end-to-end. 
See also: https://geonetwork.github.io/geonetwork-ui/main/docs/guide/dev-environment.html#backend-services
I don't think you'll already find fitting date in there, as the territories are a custom use case for MEL. That's why I think your fixture are a good alternative in this case. Otherwise, we'd need to set up a separate backend in MEL for testing (for custom uses cases).